### PR TITLE
Adding Delete Node sample

### PR DESCRIPTION
**Pull request by tonypnode**
Adding an example of querying for the Uri and deleting a node.

### CODE REVIEW

File: samples/delete_node.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+password = ''
+
+# disable SSL warnings
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+# Setup the connection
+swis = SwisClient(npm_server, username, password)
+
+# Find the Uri you want to delete based on a SWQL query
+results = swis.query("SELECT IPAddress, Caption, Uri FROM Orion.Nodes WHERE IPAddress LIKE '127.0.0.6'")
```
Comment (tdanner) [inline samples/delete_node.py]: Please change the query to extract `'127.0.0.6'` to a query parameter.

**Comment by tonypnode** [comment]
There you go! Fairly certain that's what you were looking for...

File: samples/delete_node.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+password = ''
+
+# disable SSL warnings
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+# Setup the swis connection
+swis = SwisClient(npm_server, username, password)
+
+# Find the Uri you want to delete based on a SWQL query
+results = swis.query("SELECT IPAddress, Caption, Uri FROM Orion.Nodes WHERE IPAddress = @ip_addr", ip_addr='127.0.0.6')
```
Comment (tonypnode) [inline samples/delete_node.py]: I'm assuming this is what you were looking for...

**Comment by tdanner** [comment]
Thanks!

**Pull request merged**